apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -24,8 +24,30 @@ export default class TutorialSet extends React.Component {
   tutorialClicked = item =>
     this.setState({showingDetail: true, chosenItem: item});
 
-  tutorialDetailClosed = () =>
+  tutorialDetailClosed = () => {
+    // okay, so while the tutorialDetail window is open, it's possible that the focus has changed
+    // the user could've hit left/right arrows while the detail was open and moved focus, or also could've
```

Comment:
Super tiny NIT: 
`okay` -> `Okay` and `has changed // the user` -> `has changed. // The user`
Just a norm here. The reason behind it: starting sentences with capital letters and ending with periods makes reading and understanding comments just a little easier.

Reply:
done!